itou/utils/enums_context_processors.py

```diff
@@ -0,0 +1,14 @@
+import itou.job_applications.enums as job_aplication_enums
+import itou.users.enums as users_enums
+
+
+def expose_enums(request):
+    """
+    Put things into the context to make them available in templates.
+    https://docs.djangoproject.com/en/2.1/ref/templates/api/#using-requestcontext
```

Comment:
Allez, utilisons une doc à jour ;)
```suggestion
    https://docs.djangoproject.com/en/4.1/ref/templates/api/#using-requestcontext
```